.asf.yaml

```diff
@@ -41,6 +41,8 @@ github:
     main:
       required_pull_request_reviews:
         require_code_owner_reviews: false
+        dismiss_stale_reviews: true
```

Comment:
sorry, @jbonofre , I could not find any description for this option in the README linked from the PR :sweat_smile: Could you add a comment to explain how it works?

Reply:
Thx!

Reply:
So, IIUC, `dismiss_stale_reviews` will automatically dismiss approvals _and_ "request changes" when new commits are added?

Reply:
I agree that's a little awkward... but we already have the option to manually do that dismissal, so automating it doesn't really move the needle too much imo.

Reply:
My understanding is that an approval will be still valid. I propose to merge and try 😄 If it's worse than before, I will revert.